docker/docker-compose.yml

```diff
@@ -320,6 +320,10 @@ services:
   setup:
     image: "imlteam/manager-setup:5.1.1-dev"
     command: ["tail", "-f", "/dev/null"]
```

Comment:
I wonder if any  command is needed. Even if it is, could be "`true`" ?

Reply:
We might be able to change it but I'd rather do it in a separate patch if that's ok.